benchmark_runner/common/oc/oc_exceptions.py

```diff
@@ -164,3 +164,10 @@ class ODFHealthCheckTimeout(OCError):
     def __init__(self):
         self.message = f"ODF health check timeout"
         super(ODFHealthCheckTimeout, self).__init__(self.message)
+
+
+class NodeNotReady(OCError):
+    """This exception return odf healthcheck timeout error"""
```

Comment:
"returns"

(yeah, have to like how an "s" suffix in English is normally the plural for nouns but the singular for verbs and vice versa)